docs/dashboard/catalog/products.md

```diff
@@ -366,6 +366,10 @@ To track inventory, you need to enable the _Track Inventory_ checkbox.
 
 You can repeat this process for all sizes or types of the same product by clicking the Variants card on the left side. Each variant has a specific SKU code (Stock Keep Unit) which is unique to specific variants.
 
+Once the product availability is set on the main product page, you will also find information as to which channel a specific variant of a product is assigned to at the top of each variant page.
+
+![Editing product variants](../screenshots/catalog-product-variant-details.png)
```

Comment:
This screenshot should be updated as well but I think we're using the old one here? A new section was added:
![image](https://user-images.githubusercontent.com/5421321/118796749-2e87e900-b89c-11eb-818f-9472a35b2b78.png)

Reply:
@maarcingebala I'm working on the master staging and that's the only new section I see on the top of the variant page.